build/Makefile

```diff
@@ -1,409 +1,420 @@
-		--PFARCH=$(PF_ARCH) \
-		--PFDISTRO=$(PF_DISTRO) \
-		--PFMAJOR=$(PF_MAJOR) \
-		--PFMINOR=$(PF_MINOR) \
-		--VERSION=$(CONTAINER_BUILDVERSION_MAJOR).$(CONTAINER_BUILDVERSION_MINOR).$(CONTAINER_BUILDVERSION_PATCH) \
-		--RELEASE=$(CONTAINER_BUILDVERSION_BUILDNR) \
-		--CONTAINER_BUILD_LIBRARY=$(CONTAINERLIB_FILENAME) \
-		--OUTPUTFILE=$(OUTPUT_PACKAGE_PREFIX) \
-		--DATAFILE_PATH=$(BASE_DIR)/installer/datafiles \
-		base_container.data linux.data linux_rpm.data
-
-	sudo $(RMDIR) $(STAGING_DIR)
-	$(MKPATH) $(INTERMEDIATE_DIR)
-	python $(SCXPAL_DIR)/installer/InstallBuilder/installbuilder.py \
-		--BASE_DIR=$(BASE_DIR) \
-		--TARGET_DIR=$(INTERMEDIATE_DIR) \
-		--INTERMEDIATE_DIR=$(INSTALLER_TMPDIR) \
-		--STAGING_DIR=$(STAGING_DIR) \
-		--BUILD_TYPE=$(BUILD_TYPE) \
-		--BUILD_CONFIGURATION=$(BUILD_CONFIGURATION) \
-		--PFARCH=$(PF_ARCH) \
-		--PFDISTRO=$(PF_DISTRO) \
-		--PFMAJOR=$(PF_MAJOR) \
-		--PFMINOR=$(PF_MINOR) \
-		--VERSION=$(CONTAINER_BUILDVERSION_MAJOR).$(CONTAINER_BUILDVERSION_MINOR).$(CONTAINER_BUILDVERSION_PATCH) \
-		--RELEASE=$(CONTAINER_BUILDVERSION_BUILDNR) \
-		--CONTAINER_BUILD_LIBRARY=$(CONTAINERLIB_FILENAME) \
-		$(DPKG_LOCATION) \
-		--OUTPUTFILE=$(OUTPUT_PACKAGE_PREFIX) \
-		--DATAFILE_PATH=$(BASE_DIR)/installer/datafiles \
-		base_container.data linux.data linux_dpkg.data
-
-	# Strip the package extension from the package filename
-	sed -re 's/.rpm$$|.deb$$//' $(INTERMEDIATE_DIR)/package_filename > $(INTERMEDIATE_DIR)/package_file.tmp; mv $(INTERMEDIATE_DIR)/package_file.tmp $(INTERMEDIATE_DIR)/package_filename
-
-	# Build the tar file containing both .rpm and .deb packages
-	cd $(INTERMEDIATE_DIR); tar cvf $(OUTPUT_PACKAGE_PREFIX).tar $(OUTPUT_PACKAGE_PREFIX).rpm $(OUTPUT_PACKAGE_PREFIX).deb
-
-	../installer/bundle/create_bundle.sh $(PF)_$(PF_DISTRO) $(INTERMEDIATE_DIR) $(OUTPUT_PACKAGE_PREFIX)
-	# Copy the shell bundle to the target directory
-	$(MKPATH) $(TARGET_DIR)
-	cd $(INTERMEDIATE_DIR); $(COPY) `cat $(INTERMEDIATE_DIR)/package_filename`.sh $(TARGET_DIR)
-
-else
-
-	@echo "========================= Performing Building RPM and DPKG packages"
-	sudo $(RMDIR) $(STAGING_DIR)
-	$(MKPATH) $(INTERMEDIATE_DIR)
-	python $(SCXPAL_DIR)/installer/InstallBuilder/installbuilder.py \
-		--BASE_DIR=$(BASE_DIR) \
-		--TARGET_DIR=$(INTERMEDIATE_DIR) \
-		--INTERMEDIATE_DIR=$(INSTALLER_TMPDIR) \
-		--STAGING_DIR=$(STAGING_DIR) \
-		--BUILD_TYPE=$(BUILD_TYPE) \
-		--BUILD_CONFIGURATION=$(BUILD_CONFIGURATION) \
-		--PFARCH=$(PF_ARCH) \
-		--PFDISTRO=$(PF_DISTRO) \
-		--PFMAJOR=$(PF_MAJOR) \
-		--PFMINOR=$(PF_MINOR) \
-		--VERSION=$(CONTAINER_BUILDVERSION_MAJOR).$(CONTAINER_BUILDVERSION_MINOR).$(CONTAINER_BUILDVERSION_PATCH) \
-		--RELEASE=$(CONTAINER_BUILDVERSION_BUILDNR) \
-		--CONTAINER_BUILD_LIBRARY=$(CONTAINERLIB_FILENAME) \
-		$(DPKG_LOCATION) \
-		--OUTPUTFILE=$(OUTPUT_PACKAGE_PREFIX) \
-		--DATAFILE_PATH=$(BASE_DIR)/installer/datafiles \
-		base_container.data linux.data linux_dpkg.data
-
-endif
+# -*- mode: Makefile; -*-

```

Comment:
Do you know why this file is showing a fully changed ? Can you just edit the fluentbitplugin target part and few other lines so one can makeout what the actual difference is ?